openapi.yml

```diff
@@ -34,12 +46,43 @@ paths:
       security:
         - bearerAuth: []
       responses:
+        <<: *common-responses
         200:
           description: Successful response
-        401:
-          description: Unouthorized response
-        500:
-          description: Error response
+  /api/v1/profile:
```

Comment:
@alexgpeppe version should not be in the `paths`:

- all the paths should be version-consistent;
- if you release a `v2` you should be able to change `/login` and `/logout` accordingly

- a change in the login/logout may impact with `/profile` and you should be able to make them work consistently